lib/android/src/main/java/com/airbnb/android/react/maps/AirMapManager.java

```diff
@@ -220,6 +222,18 @@ public void receiveCommand(AirMapView view, int commandId, @Nullable ReadableArr
         view.animateToCoordinate(new LatLng(lat, lng), duration);
         break;
 
+      case ANIMATE_TO_VIEWING_ANGLE:
+        angle = (float)args.getDouble(0);
```

Comment:
I do not like the casting to float. I wonder if you guys know a better way of getting a float argument? Appreciate any help :)

Reply:
This isn't a huge deal. The cast has to happen at some point. There is usually a space after a type cast though.